lib/api/fields/class-beans-fields.php

```diff
@@ -1,59 +1,69 @@
 <?php
+/**
+ * This class handles the Beans Fields workflow.
+ *
+ * @package Beans\Framework\API\Fields
+ *
+ * @since   1.0.0
+ */
+
 /**
  * Handles the Beans Fields workflow.
```

Comment:
Give a bit of context here - what is a "Beans Field"? What is a "Beans Fields workflow"?

"Field" is an ambiguous term, so be clear about the context this relates to.